packages/core/index.d.ts

```diff
@@ -173,7 +173,7 @@ declare module '@rjsf/core' {
         className: string;
         disabled: boolean;
         idSchema: IdSchema;
-        items: {
+        items?: {
```

Comment:
Why should this be optional?

Reply:
may be we missed something here, let me check